### PR TITLE
Orre Colosseum Sprites and fixes

### PR DESCRIPTION
**Pull request by Farboo171**
## Description
Provides a new batch of Orre sprites, as well as fixing a few errors found during testing

## Motivation and Context
Gotta fill in the art and fix the errors. Hiding trainers that will not be there at certain times, and fixing a mixed-up trainer party.

## How Has This Been Tested?
checked a few sprites, checked a before-and-after hide for some trainers that should be hidden

## Types of changes

- Orre Support

### CODE REVIEW

**Comment by osherwott** [comment]
Why are there duplicate Cipher Admin sprites?

**Comment by Farboo171** [comment]
> Why are there duplicate Cipher Admin sprites?

There's no good way to name them such that the display names work well and the correct images are referenced in dungeons, near as I can tell.

File: src/scripts/dungeons/Dungeon.ts [hunk 1]
```diff
@@ -4592,31 +4599,31 @@ dungeonList['Realgam Tower Battles'] = new Dungeon('Realgam Tower Battles',
                 new GymPokemon('Rhydon', 1100000, 50),
                 new GymPokemon('Starmie', 1100000, 49),
                 new GymPokemon('Manectric', 1100000, 50),
-            ], { weight: 1, requirement: new QuestLineCompletedRequirement('Shadows in the Desert', GameConstants.AchievementOption.less) }, 'Ein', '(ein)'),
+            ], { weight: 1, hide: true, requirement: new QuestLineCompletedRequirement('Shadows in the Desert', GameConstants.AchievementOption.less) }, 'Ein', '(ein)'),
```
Comment (osherwott) [inline src/scripts/dungeons/Dungeon.ts:4602]: Just do this for all of them
```suggestion
            ], { weight: 1, hide: true, requirement: new QuestLineCompletedRequirement('Shadows in the Desert', GameConstants.AchievementOption.less) }, 'Ein', 'Ein'),
```

Comment (Farboo171) [reply src/scripts/dungeons/Dungeon.ts]: That did it, thanks!

**Review by Jaaslet — APPROVED** [review]

**Pull request merged**